### ISSUE TITLE
Change or possibly just add-on to the movement of setups going up/down.

### ISSUE THREAD

**Issue by WaFFlzz** [dillydill123/inventory-setups#39]
Maybe something as simple as adding an option onto this list, that says "Move to top", and/or "Move to bottom", what happens with me, is i make a new setup and it gets added to the bottom of my list of ALOT of setups, and some of my setups are outdated/irrelevant at the moment, so now say i have a new setup added to the bottom of 100 setups, that i like to use more frequently than others, i have to scroll all the way to the bottom to see it, making an option to move directly to the top would fix this rather than manually right click moving up the setup 1 at a time, which takes a very long time.
TLDR, add a quick option to move a setup to the top of the list here. 
![image](https://user-images.githubusercontent.com/65757799/87114389-754d9100-c23e-11ea-9fad-14bf7c1d78aa.png)
Also, you could maybe somehow add the option to swap 2 setups places with each other. Dont know about this one though, this isnt really that necessary.

**Comment by dillydill123** [comment]
Hey, thanks for all the suggestions.

I want to revamp how setups can be moved. I have an idea to create a separate window that just contains setup names which you can drag to change the order of setups. This is the long term future.

For the short term, I can add three extra entries to setups:

Move Inventory Setup To Top - Move the setup to the top
Move Inventory Setup To Bottom - Move the setup to the bottom
Move Inventory Setup to Position ... - This will prompt a number between 0 and # of setups, and will place the setup in that position.

Hopefully this alleviates the issue in the short term.

**Comment by WaFFlzz** [comment]
that definitely does! i didnt think of the to position option, definitely would fix it :)

**Comment by WaFFlzz** [comment]
maybe add a number indicator for each setup always visible then if you do end up doing the move to position entry, so you know exactly where its going with ease

**Comment by dillydill123** [comment]
![uXpki8gMv5](https://user-images.githubusercontent.com/15637422/87863257-3c40ab00-c927-11ea-9662-3e68b7afbcbc.gif)

Here is the current progress. You can tell which position an index is in by right clicking and selecting the last option. It's not as good as a number indicator but I didn't want to clutter the panels more.

**Comment by dillydill123** [comment]
additional movement options feature has been added to 1.7 and is available now.
